doc/create-announcement-article.rb

```diff
@@ -0,0 +1,336 @@
+require 'optparse'
+
+class ArticleGenerator
+  attr_reader :output_file_path
+
+  def initialize(release_date, groonga_version, groonga_previous_version, groonga_org_repository)
+    @groonga_version = groonga_version
+    @groonga_previous_version = groonga_previous_version
+    @groonga_version_in_link = groonga_version.gsub(".", "-")
```

Comment:
Do we need `groonga_` prefix  here?
This script is only for Groonga, right?

Reply:
No, `ArticleGenerator` will be reused by Mroonga/PGroonga.
I have fixed it.